mypy/semanal_enum.py

```diff
@@ -108,7 +108,8 @@ class A(enum.Enum):
             # Error. Construct dummy return value.
             info = self.build_enum_call_typeinfo(var_name, [], fullname, node.line)
         else:
-            name = cast(StrExpr, call.args[0]).value
+            assert isinstance(call.args[0], StrExpr)
```

Comment:
Might be cleaner to have `parse_enum_call_args` return this as another tuple member.